docs/index.md

```diff
@@ -5,61 +5,56 @@
   reference, ROCm, AMD">
 </head>
 
-# AMD ROCm™ documentation
+# AMD ROCm documentation
 
-Welcome to the ROCm docs home page! If you're new to ROCm, you can review the following
-resources to learn more about our products and what we support:
+ROCm is an open-source software platform optimized to extract HPC and AI workload
+performance from AMD Instinct accelerators and AMD Radeon GPUs while maintaining
+compatibility with industry software frameworks. For more information, see [What is ROCm?](./what-is-rocm.rst)
 
-* [What is ROCm?](./what-is-rocm.rst)
-* [Release notes](./about/release-notes.md)
+If you're using Radeon GPUs, consider reviewing {doc}`Radeon-specific ROCm documentation<radeon:index>`.
 
-You can install ROCm on our Radeon™, Radeon™ PRO, and Instinct™ GPUs. If you're using Radeon
-GPUs, we recommend reading the
-{doc}`Radeon-specific ROCm documentation<radeon:index>`.
+Installation instructions are available from:
 
-For hands-on applications, refer to our [ROCm blogs](https://rocm.blogs.amd.com/) site.
+* {doc}`ROCm installation for Linux<rocm-install-on-linux:index>`
+* {doc}`ROCM installation for Windows<rocm-install-on-windows:index>`
+* [Deep learning frameworks installation](./how-to/deep-learning-rocm.rst)
 
-Our documentation is organized into the following categories:
+ROCm documentation is organized into the following categories:
 
 ::::{grid} 1 2 2 2
 :class-container: rocm-doc-grid
 
 :::{grid-item-card}
 :class-card: sd-text-black
-:img-top: ./data/banner-installation.jpg
-:img-alt: Install documentation
+:img-top: ./data/banner-concepts.jpg
+:img-alt: Concepts documentation
```

Comment:
```suggestion
:img-alt: Explanation documentation
```
Do we really want to change from Conceptual -> Concepts?
Maybe we should just drop back to using "Explanation" instead, like the original quadrant:
https://diataxis.fr/